tests/performance/apdex-score/test/specs/num-1/test.apdex.js

```diff
@@ -2,24 +2,49 @@ require('dotenv').config();
 
 const loadSettings = require('../../../settings-provider');
 
-const LoadPage = require('../../pageobjects/load.page');
-const LoginPage = require('../../pageobjects/login.page');
-const PeoplePage = require('../../pageobjects/people.page');
-const TasksPage = require('../../pageobjects/tasks.page');
-const MessagesPage = require('../../pageobjects/messages.page');
-const ReportsPage = require('../../pageobjects/reports.page');
-const PerformancePage = require('../../pageobjects/performance.page');
+const LoadPage = require('../../page-objects/load.page');
+const LoginPage = require('../../page-objects/login.page');
+const contactsPage = require('../../page-objects/contacts.page');
+/*
+const TasksPage = require('../../page-objects/tasks.page');
+const MessagesPage = require('../../page-objects/messages.page');
+const ReportsPage = require('../../page-objects/reports.page');
+const PerformancePage = require('../../page-objects/performance.page');
+*/
 
 describe('Apdex Performance Workflows', () => {
+  const settingsProvider = loadSettings();
+  const REPETITIONS = settingsProvider.getIterations();
+
   before(async () => {
-    const settingsProvider = loadSettings();
     const instanceUrl = settingsProvider.getInstanceURL();
     const hasPrivacyPolicy = settingsProvider.hasPrivacyPolicy();
     const user = settingsProvider.getUser('offline', 'chw');
     await LoadPage.loadInstance(instanceUrl);
     await LoginPage.login(user.username, user.password, hasPrivacyPolicy);
   });
 
+  for (let i = 0; i < REPETITIONS; i++) {
+    it('should load contact list', async () => {
+      await contactsPage.loadContactList(settingsProvider);
+    });
+
+    it('should load CHW area', async () => {
+      await contactsPage.loadCHWArea(settingsProvider);
+    });
+
+    it('should load Household', async () => {
+      await contactsPage.loadHousehold(settingsProvider);
+    });
+
+    it('should load patient', async () => {
+      await contactsPage.loadPatient(settingsProvider);
+    });
```

Comment:
It's clearer to have specific test cases for us, since in Apdex we care about page performance, so better have it by page.